upb/pb/encoder.c

```diff
@@ -374,6 +374,11 @@ static void *encode_startdelimfield(void *c, const void *hd) {
   return ok ? c : UPB_BREAK;
 }
 
+bool encode_unknown(void *c, const void *hd, const char *buf, size_t len) {
```

Comment:
I think this should be static and not declared in .h?

Reply:
This needs to be exposed to php extension to implement custom handler.